concepts/segment.md

```diff
@@ -0,0 +1,18 @@
+---
+grand_parent: Resources
+parent: Concepts
+title: Segment
+description: A sequence of characters
+---
+
+A segment is a unit that is part of a document or other file that is processed by a Computer-Aided Translation tool (CAT).
+Segments paired into source-target units are building blocks of translation memory. 
+Splitting text into such units allows to assign matches from translation memory and to identify repetitions.
```

Comment:
Instead of "assign", perhaps we could say "apply".

Reply:
Done